.github/workflows/lint.yml

```diff
@@ -0,0 +1,22 @@
+# This workflow will do a clean install of node dependencies, build the source code and run lint across different versions of node
+name: Lint CI
+
+on:
+  [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: [ubuntu-latest, macos-latest, ubuntu-16.04]
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
```

Comment:
It's a real overkill to _lint_ on 9 platforms. Just run on Ununtu latest with the Node LTS (12.x) - we are not testing ESLint here, we linting our code.